developer/src/kmldmlc/build.sh

```diff
@@ -46,6 +46,8 @@ fi
 
 if builder_has_action build; then
   npm run build
+  # We need the schema file at runtime and bundled
+  cp "$KEYMAN_ROOT/resources/standards-data/ldml-keyboards/techpreview/ldml-keyboard.schema.json" "$THIS_SCRIPT_PATH/build/src/"
```

Comment:
We should move this to the configure step and then use this file also in test/helpers/index.ts -- keeps things neat and tidy and avoids deeper cross-referencing. But will do this in a follow-up PR.